CHANGELOG.md

```diff
@@ -21,8 +21,10 @@ Changes are grouped as follows
 
 ## [Unreleased]
 
+## [1.3.0] - 2019-10-09
```

Comment:
1.3.1*

Reply:
Lawl, fixed this the first time 😅 On it!